charts/nango/templates/server/server-deployment.yaml

```diff
@@ -62,6 +62,8 @@ spec:
                 secretKeyRef:
                   name: nango-secrets
                   key: mailgun-api-key
+            - name: SERVER_PORT
```

Comment:
We need to set the default value in the `values.yaml` helm chart itself - otherwise the `SERVER_PORT` value gets a default here but _not_ the other two attributes where it was/is required.